release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -209,6 +209,26 @@ SR-IOV support is now available for xref:../networking/hardware_networks/about-s
 ==== {product-title} CIDR Ranges for Networks
 Be aware that CIDR ranges for networks are not adjustable after cluster installation. Red Hat does not provide direct guidance on determining the range because it requires careful consideration of the number of pods created.
 
+[id="ocp-4-11-new-metallb-CRDs-logging-control"]
+==== Support for new MetalLB custom resource defintions (CRDs) and control over logging verbosity
+
+MetalLB CRDs have been updated with some new CRDs to support more complex configuration. 
+
+The following new CRDs have been added: 
+
+* IPAddressPools
+* L2Advertisement
+* BgpAdvertisement
```

Comment:
communities is another CRD introduced.

The ip address pool are no longer tied to the mode used for LoadBalancer services, it is combined pool for both modes.

Reply:
@kquinn1204 Could you amend the node to say "The IP Address Pool can be advertised using either L2Advertisement or BGPAdvertisement in contrast to IPAddressPool that could either be 'layer2' or 'bgp'".